docs/source/configuration/entity-caching.mdx

```diff
@@ -129,9 +128,164 @@ This entry contains an object with the `all` field to affect all subgraph reques
       "data": "key2"
     }
 }
+```
+
+### Entity cache invalidation
+
+When existing cache entries need to be replaced, the router supports a couple of ways for you to invalidate entity cache entries:
+- **Invalidation endpoint** - the router exposes an invalidation endpoint that can receive invalidation requests from any authorized service. This is primarily intended as an alternative to the extensions mechanism described below. For example a subgraph could use it to trigger invalidation events "out of band" from any requests received by the router or a platform operator could use it to invalidate cache entries in response to events which aren't directly related to a router.
```

Comment:
can this link to the `Invalidation HTTP Endpoint` header? As I was reading through this, I kept thinking this endpoint is done through regular `router.yaml`, i.e. the section that follows right after this.

Reply:
done in 4cd883cbf